spec/features/notes_on_merge_requests_spec.rb

```diff
@@ -132,69 +141,75 @@
 
     subject { page }
 
-    describe "when adding a note" do
+    describe 'when adding a note' do
       before do
         click_diff_line
       end
 
-      describe "the notes holder" do
-        it { should have_css(".js-temp-notes-holder") }
+      describe 'the notes holder' do
+        it { is_expected.to have_css('.js-temp-notes-holder') }
 
-        it { within(".js-temp-notes-holder") { should have_css(".new_note") } }
+        it 'has .new_note css class' do
+          within('.js-temp-notes-holder') do
+            expect(subject).to have_css('.new_note')
+          end
+        end
       end
 
-      describe "the note form" do
+      describe 'the note form' do
         it "shouldn't add a second form for same row" do
           click_diff_line
 
-          should have_css("tr[id='#{line_code}'] + .js-temp-notes-holder form", count: 1)
+          is_expected.
+            to have_css("tr[id='#{line_code}'] + .js-temp-notes-holder form",
+                        count: 1)
         end
 
-        it "should be removed when canceled" do
+        it 'should be removed when canceled' do
           within(".diff-file form[rel$='#{line_code}']") do
-            find(".js-close-discussion-note-form").trigger("click")
+            find('.js-close-discussion-note-form').trigger("click")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.